README.md

```diff
@@ -8,16 +10,20 @@ cloud (e.g. GCP) environment.
 To install Cromwell, you can use conda:
 
 ```
-create -n cromwell -c conda-forge cromwell
+conda create -n cromwell -c conda-forge cromwell
```

Comment:
😆 Good job, I meant to add this in my previous PR too but somehow forgot..

Reply:
Ah I never quite realised the difference, `gsutil` is for storage, and `gcloud` is for the rest ([source](https://stackoverflow.com/a/63006519/2860731))